src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/GeneratedStatements.cs

```diff
@@ -22,6 +22,8 @@ public struct GeneratedStatements
         public ImmutableArray<StatementSyntax> NotifyForSuccessfulInvoke { get; init; }
         public ImmutableArray<StatementSyntax> GuaranteedUnmarshal { get; init; }
         public ImmutableArray<StatementSyntax> Cleanup { get; init; }
+        //public ImmutableArray<StatementSyntax> CleanupNativeOut { get; init; }
```

Comment:
What are the next steps for commented out code? At a minimum we need to have that captured here so we know how to uncomment or delete it.